docs/source/building-applications.rst

```diff
@@ -0,0 +1,22 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Building Applications
```

Comment:
I think that just kills the structure again. What should work I believe is have a single overview page with all of these titles and the toctrees which at least cleans things up a bit and maybe we can find a way to make that page itself unaccessible.

Reply:
I tried making a daml-guide-index page but it's not working. It pulls the guide but flattens it out. Canton already had an index in there, which is also flattened out if you call to it from the assembly index. Here's what I'm seeing in case I'm missing something. 
https://user-images.githubusercontent.com/42810081/153503734-b291823a-b0a1-4b15-8259-ce4877bc1cdc.mov

Reply:
@moritzkiefer-da adding intro content to the currently empty pages is the only option. I wasn't able to have everything in a single toc file and wrote this as a note to self:

> The main ToC is located in the docs->source->index.rst file and it maps the 1st (main elements such as “Getting started” or “Writing Daml”) and 2nd level (Getting started->Installing the SDK etc.) within the ToC structure. 3rd and 4th levels (if there are any, e.g., “Writing Daml->An introduction to Daml->Basic contracts” etc.) are mapped then within the 2nd level pages that have their own ::toctree: directives (The “Introduction to Daml” page that corresponds to “docs->source->daml->intro->0_intro.rst has its own ToC ::toctree: directive at the bottom of the rst file)

Reply:
We could have an index on the pages by removing the `:hidden:` tag. That will at least populate the page with a table of contents. We can definitely have Carrie help us write content for those pages as well, but that'll have to happen after the Getting Started Guide is in. 

Can we merge this in with the table of contents on the empty pages for now?
<img width="1605" alt="Screen Shot 2022-02-11 at 3 29 16 PM" src="https://user-images.githubusercontent.com/42810081/153665629-6610fc9f-b169-4614-8edc-4d84152ea970.png">

Reply:
Alright, let’s go with this for now. I think we can potentially also change how we generate the toc. Afaict, that you need titles after the first level is not a sphinx limitation but a limitation of our theme.